homework/content.js

```diff
@@ -27,11 +23,19 @@
       p.textContent = passage;
       buff.appendChild(p);
     });
-    _wrapper.appendChild(h);
-    _wrapper.appendChild(buff);
+    this._wrapper.appendChild(h);
+    this._wrapper.appendChild(buff);
+  },
+  
+  start() {
+	  this._wrapper = document.querySelector('#note-content-wrapper');
+      window.addEventListener('note-open', (function(event){
```

Comment:
You should set the `this` as the `listener` in the argument list. See the 2nd example of manual here:

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Why_use_addEventListener